Book-A-Desk.Domain/EventStore.fs

```diff
@@ -17,13 +17,17 @@ module EventsReader =
         | Some events ->
             Result.Ok events
         | None ->
-            Result.Error "could not find aggregate id"
+            match Map.count events with
```

Comment:
If we didn't have any events, we would simply error out. Now, we at least return an empty list.

This will probably not work as well once we have many aggregate ids